support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -54,9 +54,13 @@ DomainCreator:
         ExciseWithBoundaryCondition:
           DemandOutgoingCharSpeeds:
       UseLogarithmicMap: true
+    # For the envelope, setting radial distribution to logarithmic produces
+    # lower constraints than other radial distrubtions.
```

Comment:
I suggest something like "Radial distribution is logarithmic to transition from grid point spacing around the excisions to the grid point spacing in the outer shell".